src/bin/scheduler/setup/rcc.rs

```diff
@@ -176,8 +165,14 @@ fn enable_long_path_support(
     global_config: &GlobalConfig,
     plans: Vec<Plan>,
 ) -> Result<(Vec<Plan>, Vec<SetupFailure>), Cancelled> {
-    let mut command_spec =
-        RCCEnvironment::bundled_command_spec(&global_config.rcc_config.binary_path);
+    use robotmk::session::CurrentSession;
+    let session = Session::Current(CurrentSession {});
+    let mut command_spec = RCCEnvironment::bundled_command_spec(
+        &global_config.rcc_config.binary_path,
+        session
+            .robocorp_home(&global_config.robocorp_home_base)
```

Comment:
We should use `RCCEnvironment.robocorp_home` here and below. Therefore, we probably need
```
Vec<(Plan, RCCConfig)>
```
as input.

Reply:
I think this is the third time you mention it in this review. Let's pick one place for the discussion.